tribuo-classification/common/src/main/java/com/r3/conclave/sample/common/InputData.java

```diff
@@ -1,26 +1,23 @@
 package com.r3.conclave.sample.common;
 
 import org.tribuo.MutableDataset;
-
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.List;
 
 /**
  * This is the custom object containing input data which is sent by client to enclave and vice versa
  */
 public class InputData implements Serializable {
     private static final long serialVersionUID = 6529685098267757690L;
 
-    private String inputType;
+    private Role inputType;
```

Comment:
Shouldn't this property be called `role` now?

Reply:
yes changed